packages/core/admin/admin/src/pages/SettingsPage/components/Tokens/FormiTokenContainer/LifeSpanInput.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
```

Comment:
there is a typo in the component's folder name `FormiTokenContainer` instead of `FormTokenContainer`

Reply:
this file can be removed because we use the `components/Tokens/LifeSpanInput` instead

Reply:
deleted